### DISCUSSION TITLE
Crash experienced running run_tests

### DISCUSSION THREAD

**Discussion by ttnatarajan** [idaholab/moose#16602] [Q&A Getting Started] [answered]
This may be a simple problem.  I am installing MOOSE for the first time on a Windows 10 machine using CONDA and I get all the way to the test part.  When I get to testing it using the "run_tests" command, I run into the following error (see below).  I see two information: 1) libquadmath.so.0 is not being found, and 2) the test reports a crash (something about oversized).  I am not sure I completely understand the information and how to search specifically for this issue on the discussion forum to help resolve it.  Help to guide to resolve this problem or a pointer to another discussion question which may help me is appreciated.

(moose) nat1840@TC-D900675:~/projects/moose/test$ make -j 8
(moose) nat1840@TC-D900675:~/projects/moose/test$ ./run_tests -i always_ok -p 2
tests/test_harness.always_ok: Working Directory: /home/nat1840/projects/moose/test/tests/test_harness
tests/test_harness.always_ok: Running command: mpiexec -n 2 /home/nat1840/projects/moose/test/moose_test-opt -i good.i --error --error-unused --error-override --no-gdb-backtrace
tests/test_harness.always_ok: /home/nat1840/projects/moose/test/moose_test-opt: error while loading shared libraries: libquadmath.so.0: cannot open shared object file: No such file or directory
tests/test_harness.always_ok: /home/nat1840/projects/moose/test/moose_test-opt: error while loading shared libraries: libquadmath.so.0: cannot open shared object file: No such file or directory
tests/test_harness.always_ok:
tests/test_harness.always_ok:
tests/test_harness.always_ok: Exit Code: 127
tests/test_harness.always_ok: ################################################################################
tests/test_harness.always_ok: Tester failed, reason: CRASH
tests/test_harness.always_ok:
tests/test_harness.always_ok ...................................................... [OVERSIZED] FAILED (CRASH)

Final Test Results:
--------------------------------------------------------------------------------------------------------------
tests/test_harness.always_ok ...................................................... [OVERSIZED] FAILED (CRASH)
--------------------------------------------------------------------------------------------------------------
Ran 1 tests in 0.9 seconds.
0 passed, 0 skipped, 0 pending, 1 FAILED
(moose) nat1840@TC-D900675:~/projects/moose/test$

**Comment by aladshaw3** [comment, marked as answer]
I just ran into this exact same problem today. I was able to get MOOSE to build and run by manually installing gcc, which contains the libquadmath library. 

Try "sudo apt install gcc", then clean out and rebuild MOOSE (i.e., do a "make clean", then "make -j4" in the test directory). That worked for me, so it might solve your problem.

**Reply by ttnatarajan** [reply]
Thank you.  That took care of the issue.
